bundles/automation/org.eclipse.smarthome.automation.commands/OSGI-INF/AutomationCommandsPluggable.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~  Copyright (c) 2016 by ProSyst Software GmbH.,
+  ~  url: http://www.prosyst.com
+  ~  All rights reserved.
+  ~
+  ~  This software is the confidential and proprietary information
+  ~  of ProSyst Software GmbH. You shall not disclose such Confidential
+  ~  Information and shall use it only in accordance with the terms
+  ~  of the license agreement you entered into with ProSyst.
+  -->
+<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" activate="activate" deactivate="deactivate"
+               immediate="true" name="org.eclipse.smarthome.automation.commands">
+
+    <implementation class="org.eclipse.smarthome.automation.internal.commands.AutomationCommandsPluggable"/>
+
+    <service>
+        <provide interface="org.eclipse.smarthome.io.console.extensions.ConsoleCommandExtension"/>
+    </service>
+
+    <reference bind="setRuleRegistry" cardinality="0..1"
```

Comment:
Afaik, greedy/reluctant doesn't exist in OSGi 4.2 - and if reluctant is the behavior in this case, I think we have to change it to 1..1.